src/js/common/schemaform/AskVAQuestions.jsx

```diff
@@ -7,12 +7,14 @@ function AskVAQuestions(props) {
         <div className="ask-va-questions help-sidebar-box">
           <h2 className="help-h2-border help-h2">Need Help?</h2>
           {props.children}
-          <p className="talk">To report a problem with this form,<br/>
-          call the Vets.gov Technical Help Desk:</p>
+          <p className="talk">
+            To report a problem with this form,
```

Comment:
Is the `<br/>` removal deliberate? According to the design in the connected issue, it looks like the previous layout is still desired, unless this is now being achieved another way?

Reply:
I'm not sure if it's still desired. I got no indication in the copy given for this change that the longer lines were to be broken into two lines (at least not forced instead of naturally due to limited screen space) and thought it would be inconsistent to have this be split but not the others.